hivemind_core/admin/static/css/style.css

```
@@ -0,0 +1,898 @@
+            display: flex;
+            align-items: center;
+            gap: 12px;
+            min-width: 300px;
+            animation: slideIn 0.3s ease;
+        }
+
+        .toast.success { border-left: 4px solid var(--accent-success); }
+        .toast.error { border-left: 4px solid var(--accent-danger); }
+        .toast.warning { border-left: 4px solid var(--accent-warning); }
+
+        @keyframes slideIn {
```

Stylelint (17.4.0)

[error] Expected keyframe name "slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)